src/components/SidebarFilter/SidebarFilter.tsx

```diff
@@ -71,6 +72,7 @@ export const SidebarFilter = ({
               onClick={() => onTagRemove(value)}
               type="button"
             >
+              <VisuallyHidden>Remove filter&nbsp;</VisuallyHidden>
```

Comment:
Why the non-breaking space?

Reply:
I wanted to make sure there is a space between `Remove filter` and the filter name. So, to have the sentence like `Remove filter&nbsp;Biomedical science research`. Should I use space instead?

Reply:
Ah okay, I think a "normal" space is more appropriate. `&nbsp;` is used for visual reasons (when you don't want a word to break on to a new line). You may need to use string interpolation so that the linter doesn't remove the space, something like:

```
{`Remove filter `}
```

or

```
Remove filter{` `}
```